openmrs-19/src/test/java/org/motechproject/openmrs19/it/MRSPersonServiceIT.java

```diff
@@ -120,15 +142,30 @@ public void shouldUpdatePerson() throws OpenMRSException, InterruptedException {
         assertEquals(newFirstName, updated.getPreferredName().getGivenName());
         assertEquals(newMiddleName, updated.getPreferredName().getMiddleName());
         assertEquals(newLastName, updated.getPreferredName().getFamilyName());
+
         // So far OpenMRS module stores only one field of createdPerson's address, which is 'address1'.
```

Comment:
now OpenMRS module stores many fields of address, so you can adjust this test. Just assert address objects. There is a ticket to create ITs for TaskDataProvider, so then we will check if fullAddressString works correctly